h/security/acl.py

```diff
@@ -0,0 +1,17 @@
+from pyramid import security
+from pyramid.security import Allow
+
+from h.security.permissions import Permission
+
+
+class ACL:
+    @classmethod
+    def for_user(cls, user):
+        client_authority = "client_authority:{}".format(user.authority)
+
+        # auth_clients with the same authority as the user may update the user
+        yield Allow, client_authority, Permission.User.UPDATE
+        yield Allow, client_authority, Permission.User.READ
+
+        # This is for inheriting security policies... do we inherit?
```

Comment:
DO WE??  DO WE???

Reply:
Literally no idea at this point. I think not...

We don't really make use of the ancestor stuff in the contexts it's pretty much a flat arrangement, but I'm not 100% yet.